guis/common/db_classes/straw_location.py

```diff
@@ -780,12 +790,17 @@ class StrawPresent(BASE, OBJECT):
     straw = Column(Integer, ForeignKey("straw.id"))
     position = Column(Integer, ForeignKey("straw_position.id"))
     present = Column(BOOLEAN)
+    time_out = Column(Integer)
+    timestamp = Column(Integer)
 
-    def __init__(self, straw, position, present=true()):
+    def __init__(self, straw, position, present=true(), time_out=None, time_in=None, timestamp=time()):
```

Comment:
so, by default now, we pass `None` to `time_in` and `time_out`. Will this submit the same entry that it did before to the DB?

Reply:
Is timestamp needed? It's OK that timestamp reflects when the entry was added to the DB. It doesn't need to be the same as time_in or time_out.

Reply:
need to make sure that passing `None` instead of not specifying an entry at all doesn't mess with the DB triggers.